eZ/Bundle/EzPublishCoreBundle/Resources/config/slot.yml

```diff
@@ -118,3 +120,15 @@ services:
         parent: ezpublish.http_cache.signalslot.http_cache
         tags:
             - { name: ezpublish.api.slot, signal: UserService\UnAssignUserFromUserGroupSignal }
+
+    ezpublish.http_cache.signalslot.trash:
+        class: %ezpublish.http_cache.signalslot.trash.class%
+        parent: ezpublish.http_cache.signalslot.http_cache
+        tags:
+            - { name: ezpublish.api.slot, signal: TrashService\TrashSignal }
+
+    ezpublish.http_cache.signalslot.recover:
+        class: %ezpublish.http_cache.signalslot.recover.class%
```

Comment:
@andrerom: Let's use double quotes around `"%parameters%"` to avoid deprecation warnings in the future.